core/ui/PageTemplate/alerts.tid

```diff
@@ -1,8 +1,8 @@
 title: $:/core/ui/PageTemplate/alerts
 tags: $:/tags/PageTemplate
 
-<div class="tc-alerts">
+<div class="tc-alerts" role="region" aria-label="Alerts">
```

Comment:
IMO it should be alertFilter so it is easier to read. 

You did define a alertFilter but then you did not use it in the list. So code is duplicated.